### DISCUSSION TITLE
error when creating a new project, updating to 8.0 and not being able to access the login page.

### DISCUSSION THREAD

**Discussion by luongnv6** [serenity-is/Serenity#7024] [Help] [answered]
Hi everyone, does anyone know what this error is? I got this error after creating a project and updating to 8.0
![image](https://github.com/serenity-is/Serenity/assets/21300748/0fb7977c-eff0-4bad-8ddb-7e0d356f5363)

**Comment by VictorTomaili** [comment]
Check this https://github.com/serenity-is/Serenity/issues/7020 
And this https://github.com/serenity-is/Serenity/issues/7021

**Comment by volkanceylan** [comment, marked as answer]
dotnet tool update sergen and transform before running.
